Projects/Home/Sources/Home.swift

```diff
@@ -1,234 +1,114 @@
 import Apollo
+import Claims
```

Comment:
Think `Claims` or any other imports of modules at the same level will cause a problem while the CI builds. 🤔 
I had to avoid this and so used the generics for all the claims views back then